person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if of_age? || parent_permission
+  end
+
+  private
+
+  def of_age?
+    return true if is_of_age? >= 18
+  end
+end
```

Comment:
- [ ] Kindly note that you don't have any method called `is_of_age?` in your class. This function should return true if the age stored in the instance variable `@age` is >= 18. example: :point_down: 

```rb 
 def iof_age?
    return true if @age >= 18
  end
```